src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,22 @@
 package mate.academy;
 
+import mate.academy.lib.Injector;
+import mate.academy.model.Movie;
+import mate.academy.service.MovieService;
+import mate.academy.service.impl.MovieServiceImpl;
+
 public class Main {
+   private static final Injector injector = Injector.getInstance("mate.academy");
     public static void main(String[] args) {
```

Comment:
add empty line between variable and method

Reply:
done))